src/frontend/src/optimizer/plan_node/batch_seq_scan.rs

```diff
@@ -85,7 +85,13 @@ impl ToBatchProst for BatchSeqScan {
         NodeBody::RowSeqScan(RowSeqScanNode {
             table_desc: Some(CellBasedTableDesc {
                 table_id: self.logical.table_desc().table_id.into(),
-                pk: vec![], // TODO:
+                pk: self
+                    .logical
+                    .table_desc()
+                    .order_desc
+                    .iter()
```

Comment:
is there an idiomatic way of doing this?

Reply:
Seems like no, probably have to implement my own: https://stackoverflow.com/questions/65328977/does-rust-implement-fromvect-for-vecu-if-i-have-already-implemented-fromt